data/10.1101/2021.06.21.449261/2021.06.21.449261.xml

```diff
@@ -355,41 +355,41 @@
-<ref id="c35"><label>35.</label><mixed-citation publication-type="website"><collab>Open collaborative writing with Manubot</collab> <string-name><given-names>Daniel S</given-names> <surname>Himmelstein</surname></string-name>, <string-name><given-names>Vincent</given-names> <surname>Rubinetti</surname></string-name>, <string-name><given-names>David R</given-names> <surname>Slochower</surname></string-name>, <string-name><given-names>Dongbo</given-names> <surname>Hu</surname></string-name>, <string-name><given-names>Venkat S</given-names> <surname>Malladi</surname></string-name>, <string-name><given-names>Casey S</given-names> <surname>Greene</surname></string-name>, <string-name><given-names>Anthony</given-names> <surname>Gitter</surname></string-name> <source>PLOS Computational Biology</source> (<year>2019</year>-<month>06</month>-<day>24</day>) <pub-id pub-id-type="doi">c7np</pub-id> DOI: <pub-id pub-id-type="doi">10.1371/journal.pcbi.1007128</pub-id> &#x00B7; PMID: <pub-id pub-id-type="pmid">31233491</pub-id> &#x00B7; PMCID: <pub-id pub-id-type="pmcid">PMC6611653</pub-id></mixed-citation></ref>
+<ref id="c1"><label>1.</label><mixed-citation publication-type="website"><string-name><given-names>Howard W</given-names><surname>French</surname></string-name><article-title>The enduring whiteness of the American media</article-title><source>The Guardian</source> (<year>2016</year>-<month>05</month>-<day>25</day>) <ext-link ext-link-type="uri" xlink:href="https://www.theguardian.com/world/2016/may/25/enduring-whiteness-of-american-journalism">https://www.theguardian.com/world/2016/may/25/enduring-whiteness-of-american-journalism</ext-link></mixed-citation></ref>

```

Comment:
@fred-atherden Ideally would we be displaying the collab in references? Obviously, we don't want undefined but is removing them the answer?

Reply:
Thanks @nlisgo, I've not removed the collabs, but instead captured them as `article-title` and moved them to after the author list. 

There are a few things here:
1. I agree we should be displaying `collab`s in references. This appears to need solving in the client - looks like the information is there in Encoda.

2. These aren't actually collabs - they are article titles. It's essentially a tagging mistake, but one that I suspect bioRxiv wouldn't want to fix. 
 
The author has used a somewhat unconventional reference style where they include the article title first and then the author (and the the source and so on). Since bioRxiv XML makes use of `mixed-citation` and their platform treats `mixed-citation` as mixed content (i.e. renders all the content in the order given), this mis-tagging is only apparent when we try to render the items where we decode the various elements in Encoda, and then the client (rather than the XML) is prescriptive about how these elements should be rendered. 

3. We will be unable to display as the author has provided, unless we somehow move to treating `mixed-citation` as mixed content which is prescriptive about how all the content should be rendered. I don't think it's a big deal here, as long as we can get the references to display all the correct information in the order prescribed by the client, but this will cause us similar (and potentially worse) problems in the future.

Reply:
It would be great if, given these points, we get this XML change added in somehow. This one is published, and the ref list looks awful as a result of these issues.